docfx.json

```
@@ -180,19 +180,82 @@
+        "docs/core/native-interop/**/**.md": "jkoritzinsky",
+        "docs/core/porting/**/**.md": "cartermp",
+        "docs/core/testing/**/**.md": "billwagner",
+        "docs/core/tools/**/**.md": "tdykstra",
+        "docs/core/tools/dotnet-**.md": "mairaw",
+        "docs/core/tutorials/**/**.md": "tdykstra",
+        "docs/core/versions/**/**.md": "billwagner",
+        "docs/core/whats-new/**/**.md": "Thraka",
         "docs/csharp/**/**.md": "billwagner",
         "docs/csharp/**/**.yml": "billwagner",
+        "docs/framework/**/**.md": "mairaw",
+        "docs/framework/app-domains/**/**.md": "gewarren",        
         "docs/framework/configure-apps/file-schema/network/**/**.md": "karelz",
         "docs/framework/configure-apps/file-schema/wcf/**/**.md": "HongGit",
+        "docs/framework/configure-apps/file-schema/winforms/**/**.md": "Thraka",
         "docs/framework/data/adonet/**/**.md": "stevestein",
         "docs/framework/data/wcf/**/**.md": "HongGit",
+        "docs/framework/deployment/**/**.md": "Thraka",
+        "docs/framework/get-started/**/**.md": "mairaw",
+        "docs/framework/install/**/**.md": "Thraka",
+        "docs/framework/migration-guide/**/**.md": "gewarren",
+        "docs/framework/misc/**/**.md": "mairaw",
+        "docs/framework/net-native/**/**.md": "mairaw",
         "docs/framework/network-programming/**/**.md": "karelz",
+        "docs/framework/performance/**/**.md": "mairaw",
+        "docs/framework/reflection-and-codedom/**/**.md": "Thraka",
+        "docs/framework/resources/**/**.md": "Thraka",
+        "docs/framework/security/**/**.md": "mairaw",
+        "docs/framework/tools/**/**.md": "tdykstra",
+        "docs/framework/ui-automation/**/**.md": "Thraka",
+        "docs/framework/unmanaged-api/alink/**/**.md": "jeffschwMSFT",
+        "docs/framework/unmanaged-api/debugging/**/**.md": "tommcdon",
+        "docs/framework/unmanaged-api/diagnostics/**/**.md": "tommcdon",
+        "docs/framework/unmanaged-api/fusion/**/**.md": "jeffschwMSFT",
+        "docs/framework/unmanaged-api/hosting/**/**.md": "jeffschwMSFT",
+        "docs/framework/unmanaged-api/profiling/**/**.md": "tommcdon",
+        "docs/framework/unmanaged-api/strong-naming/**/**.md": "jeffschwMSFT",
+        "docs/framework/unmanaged-api/tlbexp/**/**.md": "jeffschwMSFT",
+        "docs/framework/whats-new/**/**.md": "gewarren",
         "docs/framework/wcf/**/**.md": "HongGit",
+        "docs/framework/winforms/**/**.md": "Thraka",
+        "docs/framework/wpf/**/**.md": "Thraka",
+        "docs/framework/xaml-services/**/**.md": "Thraka",
         "docs/fsharp/**/**.md": "cartermp",
         "docs/machine-learning/**/**.md": "natke",
-        "docs/visual-basic/**/**.md": "KathleenDollard",
-        "docs/spark/**/**.md": "mamccrea"
+        "docs/spark/**/**.md": "mamccrea",
+        "docs/standard/**/**.md": "gewarren",
+        "docs/standard/analyzers/**/**.md": "mairaw",
```

I thought this was something @gewarren was working on. (Not trying to change assignments, trying to proofread)

```suggestion
        "docs/standard/analyzers/**/**.md": "gewarren",
```

---

We can change assignments if needed. I think we'll need to do one more round of assignments review with everyone now onboarded. But that matches what we discussed previously and https://github.com/dotnet/docs/blob/master/.github/CODEOWNERS#L111.